couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -278,23 +278,25 @@ def _create_metrics(self, data, instance_state, server, tags=None):
 
         # Get bucket metrics
         for bucket_name, bucket_stats in data['buckets'].items():
-            metric_tags = list(tags)
+            metric_tags = tags[:] or []
```

Comment:
Should this be 
```suggestion
            metric_tags = [] if tags is None else tags[:]
```
instead ?